### PR TITLE
[NETBEANS-2381] JDK 12 ver string support in nbi native win launcher

### PR DESCRIPTION
**Pull request by rtaneja1**
Handled JDK 11, 12 release version string like 11+28, 12+33.
Also registry entry path has changed. 
icon.ico was still old NetBeans icon, copied from nb/ide.launcher/windows/netbeans.ico 
 
Build nlw.exe as per : http://wiki.netbeans.org/LauncherBuild
But removed -mno-cygwin arg as compilation fails if this is present.

Updated binary zip.

### CODE REVIEW

**Comment by matthiasblaesing** [comment]
Looks sane - should be squashed on merge. (I just looked at the changes and can't comment on the compilation).

**Comment by rtaneja1** [comment]
Part fix was done in this: https://github.com/apache/incubator-netbeans/pull/734

**Comment by geertjanw** [comment]
Thanks, to me it makes sense.

File: nbi/engine/native/launcher/windows/src/JavaUtils.c [hunk 1]
```diff
@@ -138,7 +139,7 @@ JavaVersion * getJavaVersionFromString(char * string, DWORD * result) {
             major = (major) * 10 + c - '0';
             if (major > 999) return vers;
             continue;
-        } else if(c=='.'){
+        } else if(c=='.' || c=='+'){
```
Comment (sdedic) [inline nbi/engine/native/launcher/windows/src/JavaUtils.c]: Why is this ?

Comment (rtaneja1) [reply nbi/engine/native/launcher/windows/src/JavaUtils.c]: Without this, method returns null for JDK 12 version string 12+33
$ java -version
java version "12" 2019-03-19
Java(TM) SE Runtime Environment (build 12+33)
Java HotSpot(TM) 64-Bit Server VM (build 12+33, mixed mode, sharing)

Comment (rtaneja1) [reply nbi/engine/native/launcher/windows/src/JavaUtils.c]: This doesn't set the build number in version, I'll handle this.

**Comment by rtaneja1** [comment]
Thanks Svata, created NETBEANS-2528 to track build number issue

**Pull request merged**